cmd/golangBuild.go

```diff
@@ -404,6 +424,29 @@ func reportGolangTestCoverage(config *golangBuildOptions, utils golangBuildUtils
 	return nil
 }
 
+func runGolangciLint(utils golangBuildUtils, lintSettings map[string]string) error {
+	var outputBuffer bytes.Buffer
+	utils.Stdout(&outputBuffer)
+
+	err := utils.RunExecutable("golangci-lint", "run", "--out-format", lintSettings["reportStyle"])
+	if err != nil && utils.GetExitCode() != 1 {
+		return fmt.Errorf("running golangci-lint failed: %w", err)
+	}
+
+	log.Entry().Infof("lint report: \n" + outputBuffer.String())
+	log.Entry().Infof("writing lint report to %s", lintSettings["reportOutputPath"])
+	err = utils.FileWrite(lintSettings["reportOutputPath"], outputBuffer.Bytes(), 0644)
+	if err != nil {
```

Comment:
Should we add test to cover this case and exitCode 1 as well?

Reply:
You’re right. 
Done.